concrete/blocks/event_list/form.php

```diff
@@ -16,29 +16,22 @@
         <label class="control-label"><?= t('Filter By Topic') ?></label>
         <div class="radio">
             <label>
-                <input type="radio" name="filterByTopic" value="none" <?php if ($filterByTopic == 'none') {
-    ?> checked <?php 
-} ?>>
-                No topic filtering.
+                <input type="radio" name="filterByTopic" value="none" <?php echo ($filterByTopic == 'none' ? 'checked' : ''); ?>>
```

Comment:
The parenthesis here is useless. Furthermore, I'd use the short echo tags.
So:
```
<input type="radio" name="filterByTopic" value="none" <?= $filterByTopic === 'none' ? 'checked' : '' ?>>
```

Reply:
Done.